tools/devops/device-tests/templates/publish-html.yml

```diff
@@ -39,4 +66,19 @@ steps:
     allowFailedBuilds: true
     path: $(System.DefaultWorkingDirectory)/Reports
 
-# TODO: We need to find out how VSDrops can serve the page. So we are doing nothing atm.
+# Use the cmdlet to post a new summary comment. The cmdlet checks if we have the TestSummary.md file or not. It will also add the appropiate links to the commnet. 
```

Comment:
```suggestion
# Use the cmdlet to post a new summary comment. The cmdlet checks if we have the TestSummary.md file or not. It will also add the appropriate links to the commnet. 
```